source/Calamari/Program.cs

```diff
@@ -96,5 +98,17 @@ private int PrintHelp(string action)
             helpCommand.HelpWasAskedFor = false;
             return helpCommand.Execute(new[] { action });
         }
+
+        public static void EnableTLS12()
```

Comment:
Nice! Do you see any value in making this more consistent with Tentacle, Octopus Server, and friends?

https://github.com/OctopusDeploy/OctopusTentacle/blob/master/source/Octopus.Tentacle/Program.cs#L31